### PR TITLE
Remove the broken AVRO_FILE functionality.

### PR DESCRIPTION
**Pull request by big-andy-coates**
### Description 
Remove the broken AVRO_FILE functionality from the next release, (6.0).

This functionality allows a user to supply the path to an avro file in the with clause, rather than relying on the Schema Registry.  However, this no longer actually works. Hence we're removing it.

Reviewers notes:
- main purpose of PR is to remove functionality around `DdlConfig.AVRO_SCHEMA` and `DdlConfig. AVRO_SCHEMA_FILE`.
- while doing so, I moved some code from `CmdUtil` into the one class that used it: `PropertiesListTableBuilder`. Please bare in mind this is a pick-n-shift, i.e. review the fact its moved, not the contents of the code.
- removed `TestCatalog.json` and `avro_order_schema.avro` which look to no longer be used in the repo nor the system tests.

### Testing done 
The usual.

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksql-cli/src/main/java/io/confluent/ksql/cli/console/table/builder/PropertiesListTableBuilder.java [hunk 1]
```diff
@@ -33,15 +34,51 @@
...
-        .sorted(Comparator.comparing(PropertyDef::getPropertyName))
-        .map(def ->
-            ImmutableList.of(def.getPropertyName(), def.getOverrideType(), def.getEffectiveValue()))
+        .sorted(Comparator.comparing(propertyDef -> propertyDef.propertyName))
+        .map(
+            def -> ImmutableList.of(def.propertyName, def.overrideType, def.effectiveValue))
         .collect(Collectors.toList());
   }
+
+  private static List<PropertyDef> propertiesListWithOverrides(final PropertiesList properties) {
```
Comment (big-andy-coates) [inline ksql-cli/src/main/java/io/confluent/ksql/cli/console/table/builder/PropertiesListTableBuilder.java]: moved from `CmdUtils` as its only used in this one place.

**Comment by apurvam** [comment]
Note that the next release version hasn't been confirmed. It may be 5.3. If so, I am not sure that merging this change would be appropriate in a point release.

**Comment by big-andy-coates** [comment]
@apurvam, Hojjat said the functionality doesn't work at the moment anyway, so it should be fine to remove.  

Your call.

**Comment by apurvam** [comment]
I'm fine with merging this into 5.2.x if it doesn't work anyway and thus there is no real behavioral change introduced with this PR.

**Comment by big-andy-coates** [comment]
I've not actually tested if it was broken or not. I think, at the right that someone managed to get this functionality working, we should leave this until the next major version bump...

Costs us nothing...

**Comment by apurvam** [comment]
I'm fine with merging into 5.2. This functionality was replaced by the avro + schema registry support anyway. We never documented it either.

**Pull request merged**